static/app/views/replays/detail/useVirtualizedInspector.tsx

```diff
@@ -38,7 +33,6 @@ export default function useVirtualizedInspector({cache, listRef, expandPathsRef}
         }
         expandPathsRef.current?.set(index, rowState);
         handleDimensionChange(index);
-        event.stopPropagation();
```

Comment:
Does this line need to move up into the `onClick` callbacks themselves? it should be moved somewhere i think

Reply:
Updated into onClick!